pkg/config/defaults/defaults.go

```diff
@@ -0,0 +1,54 @@
+package defaults
+
+import (
+	"path"
+)
+
+const (
+	// OpenShiftBaseName is the default base name for the OpenShift product.
+	OpenShiftBaseName = "openshift.local"
+	// AtomicBaseName is the default base name for the Atomic product.
+	AtomicBaseName = "atomic.local"
+)
+
+// Defaults provides utility for creating common directory names based on the base path
+// which may be a system default or an overrridden value.  See NewNewDefaultsFromBase.
+type Defaults struct {
```

Comment:
Why do we even need to distinguish between atomic and openshift?  Just use the "origin" directory as the default and have the installer lay down those.  We don't read from a directory by default so the installer laying down /etc/atomic and then pointing to it is fine.